packages/office-ui-fabric-react/src/components/Pivot/Pivot.scss

```diff
@@ -7,233 +7,231 @@
+  border: 0;
+
+  &:hover {
+    cursor: pointer;
   }
 
-  .ms-Pivot-links {
-    font-size: 0; // Bring inline-block children together
-    height: 40px;
-    list-style-type: none;
-    padding: 0;
-    white-space: nowrap;
+  &:focus {
+    outline: none;
   }
 
-  .ms-Pivot-link {
-    color: $ms-color-neutralPrimary;
-    display: inline-block;
-    font-size: $ms-font-size-m;
-    font-weight: $ms-font-weight-regular;
-    line-height: 40px;
-    @include margin-right(8px);
-    padding: 0 8px;
-    text-align: center;
-    position: relative;
+  // Underline, not yet visible
+  &::before {
     background-color: transparent;
-    border: 0;
-
-    &:hover {
-      cursor: pointer;
-    }
-
-    &:focus {
-      outline: none;
-    }
-
-    // Underline, not yet visible
-    &::before {
-      background-color: transparent;
-      bottom: 0;
-      content: '';
-      height: 2px;
-      left: 8px;
-      position: absolute;
-      right: 8px;
-      transition: background-color $ms-duration2 $ms-ease2;
-    }
-
-    // Make room for the heavier text of the selected state
-    &::after {
-      color: transparent;
-      content: attr(title);
-      display: block;
-      font-weight: bold;
-      height: 1px;
-      overflow: hidden;
-      visibility: hidden;
-    }
+    bottom: 0;
+    content: '';
+    height: 2px;
+    left: 8px;
+    position: absolute;
+    right: 8px;
+    transition: background-color $ms-duration2 $ms-ease2;
+  }
 
-    .ms-Pivot-text,
-    .ms-Pivot-count {
-      display: inline-block;
+  // Make room for the heavier text of the selected state
+  &::after {
+    color: transparent;
+    content: attr(title);
+    display: block;
+    font-weight: bold;
+    height: 1px;
+    overflow: hidden;
+    visibility: hidden;
+  }
 
-      vertical-align: top;
-    }
+  .text,
+  .count {
+    display: inline-block;
 
-    // Spacing between icon and text/count
-    .ms-Pivot-icon + .ms-Pivot-text {
-      @include margin-left(4px);
-    }
+    vertical-align: top;
+  }
 
-    // Spacing between text and count
-    .ms-Pivot-count {
-      @include margin-left(4px);
-    }
+  // Spacing between icon and text/count
+  .icon + .text {
+    @include margin-left(4px);
+  }
 
-    //== State: Selected
-    &.is-selected {
-      font-weight: $ms-font-weight-semibold;
+  // Spacing between text and count
+  .count {
+    @include margin-left(4px);
+  }
 
-      // Show the underline
-      &::before {
-        background-color: $ms-color-themePrimary;
-      }
-    }
+  //== State: Selected
+  &.isSelected {
+    font-weight: $ms-font-weight-semibold;
 
-    //== State: Disabled
-    &.is-disabled {
-      color: $ms-color-neutralTertiary;
+    // Show the underline
+    &::before {
+      background-color: $ms-color-themePrimary;
     }
   }
 
-  .ms-Fabric.is-focusVisible .ms-Pivot-link {
-    &:focus {
-      outline: 1px solid $ms-color-neutralSecondaryAlt;
-    }
+  //== State: Disabled
+  &.isDisabled {
```

Comment:
Currently I'm following:
• Replace all .is-foo with .isFoo
• Replace all .ms-Component—variant with .isRootVariant
I am wondering how to name the module css... It would be great if you could provide some general guidance for the naming.